python/chronos/docker/chronos_unitest_docker/README.md

```diff
@@ -0,0 +1,59 @@
+# Chronos nightly build docker image
+This dockerfile helps user to build a docker image where Chronos nightly build version is deploied.
+
+## Build an image
+First `cd` to this directory. Then build your docker image with Dockerfile:
+```bash
+sudo docker build -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+```
+Or build with a proxy:
+```bash
+# typically, you need a proxy for building since there will be some downloading.
+sudo docker build \
+    --build-arg http_proxy=http://<your_proxy_ip>:<your_proxy_port> \
+    --build-arg https_proxy=http://<your_proxy_ip>:<your_proxy_port> \
+    -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+```
+According to your network status, this building will cost **around 10 mins**
+
+## Run the image
+```bash
+sudo docker run -it --rm --net=host chronos-nightly:b1 bash
+```
+
+## Use Chronos
+A conda environment is created for you automatically. `bigdl-chronos` and all of its depenencies are installed inside this environment.
+```bash
+(chronos) root@cpx-3:/opt/work#
+```
+
+## Run Unitest Examples on Jupyter Notebook for a quick use
+There have been some Chronos unitest examples about time-series passed in the docker. You can run these on Jupyter Notebook on single node server if you pursue a quick use on Chronos.
```

Comment:
remove "There have been some Chronos unitest examples about time-series passed in the docker."